app/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/CGTPropertyDisposalsConnector.scala

```diff
@@ -28,8 +30,9 @@ import scala.concurrent.{ExecutionContext, Future}
 @ImplementedBy(classOf[CGTPropertyDisposalsConnectorImpl])
 trait CGTPropertyDisposalsConnector {
 
-  def getBusinessPartnerRecord(nino: NINO)(implicit hc: HeaderCarrier): Future[HttpResponse]
+  def getBusinessPartnerRecord(nino: NINO)(implicit hc: HeaderCarrier): EitherT[Future, Error, HttpResponse]
 
```

Comment:
Should this method be as part of this trait? From my current understanding, this connector should just be responsible for submission to DES.

Reply:
that's what the signature of this function is describing: it takes in a NINO and gives back a http response. When we inject this trait into other classes, this is the functionality which is expected of it. The trait itself doesn't expose the fact that has anything to do with DES since nothing outside of it really needs to know.

Reply:
Yes, I see that now. The service implementation was further down so I didn't spot that. All good.